sn_protocol/src/messages/register.rs

```diff
@@ -22,14 +17,10 @@ use xor_name::XorName;
 pub enum RegisterCmd {
     /// Create a new [`Register`] on the network.
```

Comment:
Just a tiny suggestion, we might want to change the references from `Register` -> `SignedRegister`.
```suggestion
    /// Create a new [`SignedRegister`] on the network.
```

Also, the doc `/// A [Register] cmd that is stored in a log on Adults.` is outdated, I think.